test/unittest/unittest_nntrainer_lazy_tensor.cpp

```diff
@@ -33,37 +33,11 @@ class nntrainer_LazyTensorOpsTest : public ::testing::Test {
   /**
    * @brief return a tensor filled with contant value
    */
-  nntrainer::Tensor constant(float value) {
+  nntrainer::Tensor constant_(float value) {
```

Comment:
Q. when are you using `XXX_` ?

Reply:
I moved `nntrainer::Tensor constant(float value, unsigned int batch, ...) ` out of the fixture to `test_utils` function.

After that, `EXPECTED_*` macro was not able to resolve the overloaded function between `constant` from `test_utils` and `constant` from the fixture. While there might be an elegant way of resolve this, I just chose to differentiate the name to save time. The function is only used in the fixture environment. 

Suggestion are welcomed.